src/Microsoft.DotNet.ProjectJsonMigration/MigrationError.cs

```diff
@@ -3,6 +3,8 @@
 
 namespace Microsoft.DotNet.ProjectJsonMigration
 {
+    /// Any changes to this need to be reflected in roslyn-project-system
+    // https://github.com/dotnet/roslyn-project-system/blob/master/src/Microsoft.VisualStudio.ProjectSystem.CSharp.VS/ProjectSystem/VS/Xproj/MigrationError.cs
```

Comment:
It's all too easy and common for people not to read the comments before making a code change. It's pretty easy to enforce this sort of thing with a simple unit test. Serialize an instance of the class and compare it to a hard-coded JSON string. This will prevent accidental breaking changes to the serialization format.